plugin/src/automation/lift/lift.ml

```diff
@@ -258,6 +262,8 @@ let lift_case_args c env_c_b env_c to_c_typ npms nargs sigma =
        let c_args, b_args = take_split (nargs_lifted - npms) args in
        let sigma, arg_pair = pack_pair_rec env_c (List.tl c_args) sigma in
        sigma, List.append [List.hd c_args; arg_pair] b_args
+  | UnpackSigma ->
```

Comment:
For efficiency, here and elsewhere, since we define a custom eliminator, we may want to support lifting that eliminator directly rather than treating it like any other function.